make-rules/shared-macros.mk

```diff
@@ -120,17 +120,19 @@ SBINDIR =	/sbin
 LIBDIR =	/lib
 VARDIR =	/var
 KERNELDRVDIR =	/kernel/drv
+KERNELDRVDIR32 =/kernel/drv
 KERNELDRVDIR64 =/kernel/drv/$(MACH64)
-USRBINDIR =	$(USRDIR)/bin
+USRBINDIR = 	$(USRDIR)/bin
 USRBINDIR32 =	$(USRDIR)/bin/$(MACH32)
 USRBINDIR64 =	$(USRDIR)/bin/$(MACH64)
-USRSBINDIR =	$(USRDIR)/sbin
+USRSBINDIR = 	$(USRDIR)/sbin
 USRSBINDIR32 =	$(USRDIR)/sbin/$(MACH32)
 USRSBINDIR64 =	$(USRDIR)/sbin/$(MACH64)
-USRLIBDIR =	$(USRDIR)/lib
+USRLIBDIR = 	$(USRDIR)/lib
+USRLIBDIR32 =	$(USRDIR)/lib/$(MACH32)
```

Comment:
In some systems over the years I saw packages symlinks like ./32 -> ./ and
./i386 -> ./ so formally all such paths were fulfilled. This might also
allow for cleaner progress in your crusade for a primarily-64bit system
later on.

Besides, what I really wanted to add here were the *32 variables wherever
missing, so recipes could refer to *$(BITS) easily. We can strip /mach32
from value contents ;)

--
Typos courtesy of GMail on my Samsung Android

17.07.2016 7:12 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

In make-rules/shared-macros.mk
<https://github.com/OpenIndiana/oi-userland/pull/2027#discussion_r71076632>:

>  USRSBINDIR32 =	$(USRDIR)/sbin/$(MACH32)
>  USRSBINDIR64 =	$(USRDIR)/sbin/$(MACH64)
> -USRLIBDIR =	$(USRDIR)/lib
> +USRLIBDIR = 	$(USRDIR)/lib
> +USRLIBDIR32 =	$(USRDIR)/lib/$(MACH32)

crle/ld don't know about it, so it's useless

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/OpenIndiana/oi-userland/pull/2027/files/1408b3381e2921afdccf26d392a4e7905e233f67#r71076632>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABj5lE6xV6n_6PqBQVgyMAYDRIL8iJ2Bks5qWbmegaJpZM4Ij_l9>
.